src/entities/mentor/ui/mentor-feedback-card/mentor-feedback-card.module.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.card-wrapper {
+  max-width: 624px;
+  min-height: 510px;
+  padding: 40px 32px 40px 32px;
```

Comment:
```suggestion
  padding: 40px 32px;
```

Reply:
created an issue https://github.com/rolling-scopes/site/issues/734

Reply:
I have seen this issue, but I don't like this approach, because three values ​​(for example `margin: 1px 2px 3px`) blow my mind. If I have to choose, let's leave it as is